docs/UserGuide.md

```diff
@@ -34,14 +35,55 @@ StudentConnect is a solution for hassle-free team formation for students to brow
+
+2. Throughout this guide, we've provided screenshots that illustrate the expected output for each command and feature. This visual aid is designed to enhance your understanding and ensure that you can confidently anticipate the results of your actions. Below is a detailed overview of the various interface components that you'll become familiar with.
+
+<div style="text-align: center; border: 1px solid #000; padding: 10px; display: inline-block; margin: auto;">
+  <img src="images/StudentListUI.png" alt="Student List View of StudentConnect" style="margin-bottom: 10px;"/>
+  <div>Figure 1: Student List View of StudentConnect</div>
+</div>
+
+<div style="text-align: center; border: 1px solid #000; padding: 10px; display: inline-block; margin: auto;">
+  <img src="images/GroupListUI.png" alt="Group List View of StudentConnect" style="margin-bottom: 10px;"/>
+  <div>Figure 2: Group List View of StudentConnect</div>
+</div>
+
+- **Command Box**: This is where you can enter any command for execution. After typing your command, press Enter to submit. If there is a syntax error or the command is unrecognizable, the text will turn red, prompting you to correct the command according to the specified requirements before pressing Enter again.
+- **Feedback Box**: The feedback box is the area where the application communicates responses to your commands. It displays the correct command format if there is an input error, error messages when a command fails to execute, and the outcomes for commands like the `tasks` command.
+- **Student List**: The student list panel displays a roster of all students and their associated details. Executing any student-related command will result in this panel being updated to show the relevant student list.
+- **Group List**: Similar to the student list, the group list panel shows all existing groups along with their members and designated tutorials. Group-related commands will update this panel accordingly.
+- **Menu Options**: The menu options provide quick access to general tasks within the application.
+<div markdown="span" class="alert alert-primary">:exclamation: **Important:**
+To guarantee that all your changes and data are saved correctly, it is strongly advised to type the `exit` command into the Command Box
+instead of selecting `exit` from the menu options or closing the application window directly.
+This practice helps prevent data loss and ensures a proper shutdown of the application.
+</div>
+
+3. Should you encounter any terminology within the guide that is unfamiliar, you can consult the [glossary](#glossary) section below. It provides clear definitions and explanations to help you grasp the specialized language used throughout this guide.
+4. For any inquiries, the [FAQ](#faq) section is a good place to look for answers. In case your question is not addressed there, do not hesitate to contact us at [] for further assistance.
```

Comment:
is `[]` incomplete?

Reply:
yep not sure what to put

Reply:
Can put our tp github issues, then ask them to create issue to ask for assistance.

Reply:
done!